servicetalk-grpc-gradle-plugin/src/main/groovy/io/servicetalk/grpc/gradle/plugin/ServiceTalkGrpcPlugin.groovy

```diff
@@ -21,6 +21,7 @@ import org.gradle.api.InvalidUserDataException
 import org.gradle.api.Plugin
 import org.gradle.api.Project
 import org.gradle.api.Task
+import org.gradle.internal.os.OperatingSystem
```

Comment:
I am not a gradle export but just from the package name I would suspect this is not really something we should use. Is there nothing that we can use that is not in an internal package ?

Reply:
Agreed, using gradle internal APIs is not ideal but was not newly introduced by this PR. We can consider improving in a followup PR after we get the build stable again.

Reply:
actually looking at the internal code it is just using `os.name` ... let me try this quick replace for windows and not-windows.